docs/topics/Terminology.md

```diff
@@ -1,5 +0,0 @@
-# Terminology
```

Comment:
Yeah, sorry, that was a bit snarky. But I moved the clarification to `Guild.md` in the opening line. The distinction is fine, but having a separate page with just that sentence seemed a bit unnecessary.